test/index.ts

```diff
@@ -23,6 +23,12 @@ const dotnetIsolated6 = {
   response: "Hello, Test"
 }
 
+const dotnetIsolated7 = {
+  package: `${storagePath}/dotnet-isolated7-functions.zip`,
```

Comment:
I uploaded this zip with a test app. The test app is still using .NET 6 as TFM currently. Once [Update SDK for .NET 7 support](https://github.com/Azure/azure-functions-dotnet-worker/issues/935) is done, I will update the test app with the artifact produced from that work (the new SDK package to support .NET 7)

Reply:
Updated the test app to use an SDK package which handles .NET 7. So the test app is a .NET 7 function app now.